scripts/fortran_unit_testing/run_tests.py

```diff
@@ -427,7 +427,7 @@ def _main():
         "{}-DOS={} -DMACH={} -DCOMPILER={} -DDEBUG={} -DMPILIB={} -Dcompile_threaded={} -DCASEROOT={}".format(
             "" if not cmake_args else "{} ".format(cmake_args),
             os_,
-            machine,
+            machobj.get_machine_name(),
```

Comment:
If the machine argument is given, then I think this will give the same result as before; but this version works if the machine argument isn't given and needs to be inferred.